comparer.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2011 The LevelDB-Go and Pebble Authors. All rights reserved. Use
+// of this source code is governed by a BSD-style license that can be found in
+// the LICENSE file.
+
+package pebble
+
+import "github.com/petermattis/pebble/internal/base"
+
+// Compare exports the base.Compare type.
+type Compare = base.Compare
```

Comment:
Cool, reminds me of `using` declarations.

Reply:
Yep. Very similar.